src/ConfigurationsProvider.tsx

```diff
@@ -58,8 +59,10 @@ export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) =
   }, [appDesignSystem, sue, sueConfigData, sueProgress]);
 
   useEffect(() => {
-    setConfigurations(mergedConfig);
-    storageHelper.setConfigurations(mergedConfig);
+    if (JSON.stringify(configurations) !== JSON.stringify(mergedConfig)) {
```

Comment:
can you explain this in real words? its hard for me. if there is no SUE, why would we need t do something with merged configs? if there ist no SUE, we should not habe something merged but only the default configs.
with skipping requests, we receive no data. with having no data we can have a check in `mergedConfig` to return only default configs.
and if this i the case, the effect does not need to set new configs.
you know what i mean!?

Reply:
This control sets the config if there is a difference between the default config and mergedConfig. If they are the same then we use the default config. I don't want to bind this control to sue because in the future we want to be able to use config not only for sue but also for other features.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/9b73a39d97fe47fee665fac87344a4a56b2f55a3